.github/workflows/build.yml

```
@@ -89,27 +89,34 @@ jobs:
           CHISEL_VERSION: ${{ steps.build.outputs.CHISEL_VERSION }}
         run: |
           ARCHIVE_FILE=chisel_${CHISEL_VERSION}_${GOOS}_${GOARCH}.tar.gz
+          ARCHIVE_FILE_CKSUM="${ARCHIVE_FILE}.sha384"
```

```suggestion
          ARCHIVE_FILE_SHA384="${ARCHIVE_FILE}.sha384"
```

CKSUM is a different algorithm

---

Thanks, changed.